src/main/java/ClientSocketExample.java

```
@@ -0,0 +1,70 @@
+             InputStream in = socket.getInputStream();// читаємо данні з сервера
+             OutputStream out = socket.getOutputStream();//відправляємо данні з сервера
+             BufferedReader reader = new BufferedReader(new InputStreamReader(in));//створюєм обгортку для читання тексту з потоку
+             PrintWriter writer = new PrintWriter(out, true);//створюєм обгортку для запису даних
+             Scanner scanner = new Scanner(System.in)) // Сканер є сканер
+        {
+// далі без пояснень бо втомилась
+            String serverMessage = reader.readLine();
+            System.out.println(ClientMessage.MESSAGE_FROM_SERVER + serverMessage);
+
+            System.out.print(ClientMessage.MESSAGE_CLIENT);
+            String clientMessage = scanner.nextLine();
+            writer.println(clientMessage);
+
+            if (containsRussianCharacters(clientMessage)) {
+
+                serverMessage = reader.readLine();
+                System.out.println(ClientMessage.MESSAGE_FROM_SERVER + serverMessage);
+
+                for (int i = 0; i < 3; i++) {
+                    serverMessage = reader.readLine();
+                    System.out.println(serverMessage);
+                }
+
+                String answer = scanner.nextLine();
+                writer.println(answer);
+
+                serverMessage = reader.readLine();
+                System.out.println(ClientMessage.MESSAGE_FROM_SERVER + serverMessage);
+
+                if (serverMessage.startsWith("Correct")) {
+
+                    System.out.println(ClientMessage.MESSAGE_FROM_SERVER + reader.readLine());
+                } else {
+
+                    System.out.println(ClientMessage.MESSAGE_FROM_SERVER + reader.readLine());
+                }
+            }
+
+            System.out.println(ClientMessage.CONNECTED_CLOSE);
+
+        } catch (IOException e) {
+            e.printStackTrace();
```

not a good idea to suppress the exception